![collect-oss](https://github.com/HDRUK/oss/workflows/collect-oss/badge.svg) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12779-blue)
<p align="center">
  <img src="images/HDRUK_LOVE_OPEN_SOURCE.png">
</p>

# HDR UK Open Source Contributions (165)

### APPLIED ANALYTICS (3)
1. [HDRUK_AppliedML](https://github.com/MatthewWilletts/HDRUK_AppliedML): This repository contains code and supporting material for the HDRUK Summer School Course on Applied Deep Learning, August 22nd 2019 ![GitHub stars](https://img.shields.io/github/stars/MatthewWilletts/HDRUK_AppliedML?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0655-yellowgreen?style=flat-square)
2. [tpot](https://github.com/EpistasisLab/tpot): A Python Automated Machine Learning tool that optimizes machine learning pipelines using genetic programming. ![GitHub stars](https://img.shields.io/github/stars/EpistasisLab/tpot?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.50818-yellowgreen?style=flat-square)
3. [biobankAccelerometerAnalysis](https://github.com/activityMonitoring/biobankAccelerometerAnalysis): Extracting meaningful health information from large accelerometer datasets ![GitHub stars](https://img.shields.io/github/stars/activityMonitoring/biobankAccelerometerAnalysis?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.42785-yellowgreen?style=flat-square)


### BHF DATA SCIENCE CENTRE (2)
1. [4C-Initiative](https://github.com/HDRUK/4C-Initiative): 4C-Initiative ![GitHub stars](https://img.shields.io/github/stars/HDRUK/4C-Initiative?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.1368-yellowgreen?style=flat-square)
2. [Linked-EHR-England-2021](https://github.com/BHFDSC/Linked-EHR-England-2021): Publication: Linked electronic health records for research on a nationwide cohort including over 54 million people in England ![GitHub stars](https://img.shields.io/github/stars/BHFDSC/Linked-EHR-England-2021?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10868-yellowgreen?style=flat-square)


### BETTER CARE (3)
1. [collaborathon-2020-team3](https://github.com/HDRUK/collaborathon-2020-team3): HDR UK Collaborathon Team3 Submission ![GitHub stars](https://img.shields.io/github/stars/HDRUK/collaborathon-2020-team3?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05626-yellowgreen?style=flat-square)
2. [openpathology-web](https://github.com/ebmdatalab/openpathology-web): Openpathology website ![GitHub stars](https://img.shields.io/github/stars/ebmdatalab/openpathology-web?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.08676-yellowgreen?style=flat-square)
3. [openprescribing](https://github.com/ebmdatalab/openprescribing): A Django app providing a REST API and dashboards for the HSCIC's GP prescribing data ![GitHub stars](https://img.shields.io/github/stars/ebmdatalab/openprescribing?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.39051-yellowgreen?style=flat-square)


### BIOINFORMATICS (6)
1. [bedtools2](https://github.com/arq5x/bedtools2): bedtools - the swiss army knife for genome arithmetic ![GitHub stars](https://img.shields.io/github/stars/arq5x/bedtools2?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.54201-yellowgreen?style=flat-square)
2. [bwa](https://github.com/lh3/bwa): Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read alignment) ![GitHub stars](https://img.shields.io/github/stars/lh3/bwa?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.46594-yellowgreen?style=flat-square)
3. [ensembl-vep](https://github.com/Ensembl/ensembl-vep): The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants ![GitHub stars](https://img.shields.io/github/stars/Ensembl/ensembl-vep?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.48006-yellowgreen?style=flat-square)
4. [gatk](https://github.com/broadgsa/gatk): Official code repository for GATK versions 1.0 through 3.7 (core engine). For GATK 4 code, see the https://github.com/broadinstitute/gatk repository ![GitHub stars](https://img.shields.io/github/stars/broadgsa/gatk?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.2831-yellowgreen?style=flat-square)
5. [seqtk](https://github.com/lh3/seqtk): Toolkit for processing sequences in FASTA/Q formats ![GitHub stars](https://img.shields.io/github/stars/lh3/seqtk?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.39491-yellowgreen?style=flat-square)
6. [crossplatform_mGWAS](https://github.com/MRC-Epid/crossplatform_mGWAS): Cross-platform genetic discovery of small molecule products of metabolism and application to clinical outcomes Luca A. Lotta, Maik Pietzner, Isobel D. Stewart, Laura B.L. Wittemans, Chen Li, Roberto Bonelli, Johannes Raffler, Emma K. Biggs, Clare Oliver-Williams, Victoria P.W. Auyeung, Jianâ€™an Luan, Eleanor Wheeler, Ellie Paige, Praveen Surendran, Gregory A. Michelotti, Robert A. Scott, Stephen Burgess, Verena Zuber, Eleanor Sanderson, Albert Koulman, Fumiaki Imamura, Nita G. Forouhi, Kay-Tee Khaw, MacTel Consortium, Julian L. Griffin, Angela M. Wood, Gabi KastenmÃ¼ller, John Danesh, Adam S. Butterworth, Fiona M. Gribble, Frank Reimann, Melanie Bahlo, Eric Fauman, Nicholas J. Wareham, Claudia Langenberg bioRxiv 2020.02.03.932541; doi: https://doi.org/10.1101/2020.02.03.932541 ![GitHub stars](https://img.shields.io/github/stars/MRC-Epid/crossplatform_mGWAS?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02077-yellowgreen?style=flat-square)


### COHORT DISCOVERY (1)
1. [link-lite](https://github.com/biobankinguk/link-lite): HDR Link Lite - an open source lightweight version of BC-Link ![GitHub stars](https://img.shields.io/github/stars/biobankinguk/link-lite?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12398-yellowgreen?style=flat-square)


### COVID-19 (22)
1. [2019nCoV_proportion_asym](https://github.com/yangclaraliu/2019nCoV_proportion_asym):  ![GitHub stars](https://img.shields.io/github/stars/yangclaraliu/2019nCoV_proportion_asym?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07842-yellowgreen?style=flat-square)
2. [COVID19_clustersize](https://github.com/akira-endo/COVID19_clustersize): Analysis of COVID-19 transmission using cluster size distribution ![GitHub stars](https://img.shields.io/github/stars/akira-endo/COVID19_clustersize?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.08412-yellowgreen?style=flat-square)
3. [benevolentai-dat](https://github.com/BenevolentAI/benevolentai-dat): BenevolentAI's Diversity Analysis Tool (DAT) is a software package that can be used to produce demographic analysis reports given health data sets that contain for fields of age, sex, ethnicity, race and socio-economic status. For example, you might have a data about a cohort of patients and want to know how well you cover various ethnicities, age groups, sex groups and socio-economic status levels. Assuming your data sets have some of these fields, this software will help generate various views of the data to help inform your work. The DAT tool was developed as part of BenevolentAI's Diversity in Data Initiative, which aims to help improve the ways patients are represented in precision medicine. It is meant to help inspire other developers to find ways of assessing the data diversity in their current and prospective health data sets. ![GitHub stars](https://img.shields.io/github/stars/BenevolentAI/benevolentai-dat?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.1596-yellowgreen?style=flat-square)
4. [comix_covid-19-first_wave](https://github.com/jarvisc1/comix_covid-19-first_wave): Code and data from the analysis of the 1st wave of the CoMix 2020 contacts survey ![GitHub stars](https://img.shields.io/github/stars/jarvisc1/comix_covid-19-first_wave?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.09973-yellowgreen?style=flat-square)
5. [covid-mobility-tracker](https://github.com/SurgicalInformatics/covid-mobility-tracker): Generate maps from google mobility data ![GitHub stars](https://img.shields.io/github/stars/SurgicalInformatics/covid-mobility-tracker?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02684-yellowgreen?style=flat-square)
6. [covid-uk](https://github.com/cmmid/covid-uk): Scenario analyses for COVID-19 outbreak in the United Kingdom ![GitHub stars](https://img.shields.io/github/stars/cmmid/covid-uk?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.09416-yellowgreen?style=flat-square)
7. [covid19_asymptomatic_trans](https://github.com/thimotei/covid19_asymptomatic_trans):  ![GitHub stars](https://img.shields.io/github/stars/thimotei/covid19_asymptomatic_trans?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.16793-yellowgreen?style=flat-square)
8. [covid19_cases_from_deaths](https://github.com/thibautjombart/covid19_cases_from_deaths): Shiny app to infer caes from new deaths of COVID-19 ![GitHub stars](https://img.shields.io/github/stars/thibautjombart/covid19_cases_from_deaths?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.16056-yellowgreen?style=flat-square)
9. [los_review](https://github.com/esnightingale/los_review): Code used to produce the results presented in "A rapid review of evidence for hospital length of stay of COVID-19 patients" ![GitHub stars](https://img.shields.io/github/stars/esnightingale/los_review?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12986-yellowgreen?style=flat-square)
10. [screening_outbreak_delay](https://github.com/cmmid/screening_outbreak_delay):  ![GitHub stars](https://img.shields.io/github/stars/cmmid/screening_outbreak_delay?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
11. [COVID19_NPIs_vs_Rt](https://github.com/yangclaraliu/COVID19_NPIs_vs_Rt):  ![GitHub stars](https://img.shields.io/github/stars/yangclaraliu/COVID19_NPIs_vs_Rt?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05452-yellowgreen?style=flat-square)
12. [comix_uk_covid_restrictions](https://github.com/jarvisc1/comix_uk_covid_restrictions): code and data to reproduce analysis for Comix local and national restrictions paper ![GitHub stars](https://img.shields.io/github/stars/jarvisc1/comix_uk_covid_restrictions?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07556-yellowgreen?style=flat-square)
13. [transmission_inference](https://github.com/davideyre/transmission_inference):  ![GitHub stars](https://img.shields.io/github/stars/davideyre/transmission_inference?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.04786-yellowgreen?style=flat-square)
14. [ncov_measles_Kenya](https://github.com/CarolineNM/ncov_measles_Kenya): Analysis of risk of measles outbreak during COVID19 pandemic in Kenya ![GitHub stars](https://img.shields.io/github/stars/CarolineNM/ncov_measles_Kenya?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0597-yellowgreen?style=flat-square)
15. [SocialBubble](https://github.com/tsleng93/SocialBubble):  ![GitHub stars](https://img.shields.io/github/stars/tsleng93/SocialBubble?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.19824-yellowgreen?style=flat-square)
16. [nsaids-covid-research](https://github.com/opensafely/nsaids-covid-research):  ![GitHub stars](https://img.shields.io/github/stars/opensafely/nsaids-covid-research?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.25993-yellowgreen?style=flat-square)
17. [ISARIC 4C prognostic score](https://isaric4c.net/risk): The ISARIC 4C prognostic score uses data from more than 70,000 hospital admissions, we have devised a simple, robust and generalisable prognostic scoring system for hospitalised patients.
The 4C Mortality Score is a risk stratification score that predicts in-hospital mortality for hospitalised COVID-19 patients, produced by the ISARIC 4C consortium. It is intended for use by clinicians. It is designed to be easy-to-use, and require only parameters that are commonly available at hospital presentation.
For full details, see the [paper](https://www.medrxiv.org/content/10.1101/2020.07.30.20165464v1) introducing the score. This is an infographic that visualises risk, based on observed mortality among hospitalised adult COVID19 patients recruited into the ISARIC 4C study in the UK. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
18. [BSTI COVID-19 Imaging Database](https://www.bsti.org.uk/training-and-education/covid-19-bsti-imaging-database/):  ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
19. [ggquickeda](https://github.com/smouksassi/ggquickeda): ggplot2 and table1 summary statistics quick exploration of data ![GitHub stars](https://img.shields.io/github/stars/smouksassi/ggquickeda?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.36734-yellowgreen?style=flat-square)
20. [CMMID Interactive Applications](https://cmmid.github.io/visualisations):  ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
21. [covid-sim](https://github.com/mrc-ide/covid-sim): This is the COVID-19 CovidSim microsimulation model developed by the MRC Centre for Global Infectious Disease Analysis hosted at Imperial College, London. ![GitHub stars](https://img.shields.io/github/stars/mrc-ide/covid-sim?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.43522-yellowgreen?style=flat-square)
22. [COVID-19 Phenomics](http://covid19-phenomics.org/):  ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)


### GATEWAY (4)
1. [Winterfell](https://github.com/andrewhathaway/Winterfell): Generate complex, validated and extendable JSON-based forms in React. ![GitHub stars](https://img.shields.io/github/stars/andrewhathaway/Winterfell?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.26823-yellowgreen?style=flat-square)
2. [common-api](https://github.com/federated-data-sharing/common-api): OpenAPI definitions for the Federated Data Sharing Common API ![GitHub stars](https://img.shields.io/github/stars/federated-data-sharing/common-api?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.15263-yellowgreen?style=flat-square)
3. [gateway-api](https://github.com/HDRUK/gateway-api): HDR UK Gateway API ![GitHub stars](https://img.shields.io/github/stars/HDRUK/gateway-api?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.4569-yellowgreen?style=flat-square)
4. [gateway-web](https://github.com/HDRUK/gateway-web): HDR UK Gateway Web ![GitHub stars](https://img.shields.io/github/stars/HDRUK/gateway-web?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.48919-yellowgreen?style=flat-square)


### GATEWAY (MVP) (4)
1. [Gateway-Auth-Server](https://github.com/HDRUK/Gateway-Auth-Server):  ![GitHub stars](https://img.shields.io/github/stars/HDRUK/Gateway-Auth-Server?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.1382-yellowgreen?style=flat-square)
2. [Gateway-DB](https://github.com/HDRUK/Gateway-DB): Health Data Gateway - PostgreSQL Database ![GitHub stars](https://img.shields.io/github/stars/HDRUK/Gateway-DB?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10924-yellowgreen?style=flat-square)
3. [Gateway-Frontend](https://github.com/HDRUK/Gateway-Frontend): Health Data Gateway Frontend - react App ![GitHub stars](https://img.shields.io/github/stars/HDRUK/Gateway-Frontend?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12388-yellowgreen?style=flat-square)
4. [Gateway-Middleware](https://github.com/HDRUK/Gateway-Middleware): Health Data Gateway - GraphQL Apollo Middleware ![GitHub stars](https://img.shields.io/github/stars/HDRUK/Gateway-Middleware?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.14495-yellowgreen?style=flat-square)


### HDR UK (7)
1. [TRE-Survey](https://github.com/HDRUK/TRE-Survey):  ![GitHub stars](https://img.shields.io/github/stars/HDRUK/TRE-Survey?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05626-yellowgreen?style=flat-square)
2. [clinical-trials](https://github.com/HDRUK/clinical-trials):  ![GitHub stars](https://img.shields.io/github/stars/HDRUK/clinical-trials?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10614-yellowgreen?style=flat-square)
3. [covid-19](https://github.com/HDRUK/covid-19): HDR UK - Resources for ðŸ¦ COVID-19 Research ![GitHub stars](https://img.shields.io/github/stars/HDRUK/covid-19?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11429-yellowgreen?style=flat-square)
4. [datasets](https://github.com/HDRUK/datasets): HDR UK Gateway Datasets ![GitHub stars](https://img.shields.io/github/stars/HDRUK/datasets?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.28254-yellowgreen?style=flat-square)
5. [oss](https://github.com/HDRUK/oss): HDR UK OSS Contributions ![GitHub stars](https://img.shields.io/github/stars/HDRUK/oss?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.2151-yellowgreen?style=flat-square)
6. [schemata](https://github.com/HDRUK/schemata): HDR UK Schemas ![GitHub stars](https://img.shields.io/github/stars/HDRUK/schemata?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.19862-yellowgreen?style=flat-square)
7. [papers](https://github.com/HDRUK/papers): Extract of publications that mention HDR-UK ![GitHub stars](https://img.shields.io/github/stars/HDRUK/papers?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.27505-yellowgreen?style=flat-square)


### NATIONAL PHENOMICS RESOURCE (17)
1. [PGS_Catalog](https://github.com/PGScatalog/PGS_Catalog): An open database of polygenic scores and relevant metadata needed to apply and evaluate them correctly.  ![GitHub stars](https://img.shields.io/github/stars/PGScatalog/PGS_Catalog?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.26763-yellowgreen?style=flat-square)
2. [PhenomicsLibrary](https://github.com/HDRUK/PhenomicsLibrary):  ![GitHub stars](https://img.shields.io/github/stars/HDRUK/PhenomicsLibrary?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.03035-yellowgreen?style=flat-square)
3. [chronological-map-phenotypes](https://github.com/spiros/chronological-map-phenotypes): Machine-readable version of electronic health record phenotypes for Kuan V. and Denaxas S. et al. ![GitHub stars](https://img.shields.io/github/stars/spiros/chronological-map-phenotypes?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.18947-yellowgreen?style=flat-square)
4. [hdr-caliber-phenotype-library](https://github.com/spiros/hdr-caliber-phenotype-library): HDR UK National Phenotype Library ![GitHub stars](https://img.shields.io/github/stars/spiros/hdr-caliber-phenotype-library?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.27055-yellowgreen?style=flat-square)
5. [phenomenet-vp](https://github.com/bio-ontology-research-group/phenomenet-vp): A phenotype-based tool for variant prioritization in WES and WGS data ![GitHub stars](https://img.shields.io/github/stars/bio-ontology-research-group/phenomenet-vp?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.21639-yellowgreen?style=flat-square)
6. [phenotypes](https://github.com/HDRUK/phenotypes): HDR UK National Phenotype Library ![GitHub stars](https://img.shields.io/github/stars/HDRUK/phenotypes?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.1352-yellowgreen?style=flat-square)
7. [ukb-biomarker-phenotypes](https://github.com/spiros/ukb-biomarker-phenotypes): Phenotyping algorithms for common biomarkers in primary care EHR for UK Biobank ![GitHub stars](https://img.shields.io/github/stars/spiros/ukb-biomarker-phenotypes?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10021-yellowgreen?style=flat-square)
8. [CALIBER Drugdose](http://r-forge.r-project.org/R/?group_id=1598): medication dosage instructions in electronic health records are often in the form of text rather than numbers. This program is designed to convert the text into numbers for the dose, frequency, units, duration etc. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
9. [OurRisk.CoV](https://covid19-phenomics.org/PrototypeOurRiskCoV.html): We recently used large-scale samples of NHS patient data to estimate the excess risk of mortality which may be associated with coronavirus (COVID-19). We wanted to develop this research to inform simple tools to allow interaction and exploration by policymakers, researchers and the public. In the OurRisk.CoV tool, we provide our initial learning in relation to the implementation and development of policy responses to the COVID-19 pandemic. We have provided some simple tools to allow interaction and exploration.
OurRisk.CoV is initially intended for use by researchers, policy makers and those who seek to work with patient data to inform responses to the current emergency. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
10. [CALIBER health records research toolkit](http://caliberanalysis.r-forge.r-project.org/): This project comprises a set of R packages to assist in epidemiological studies using electronic health records databases.
CALIBER (http://caliberresearch.org/) is led from the Farr Institute @ London. CALIBER investigators represent a collaboration between epidemiologists, clinicians, statisticians, health informaticians and computer scientists with initial funding from the Wellcome Trust and the National Institute for Health Research.
The goal of CALIBER is to provide evidence across different stages of translation, from discovery, through evaluation to implementation where electronic health records provide new scientific opportunities. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
11. [COVID-19 Phenomics](http://covid19-phenomics.org/):  ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
12. [discrete_frechet](https://github.com/spiros/discrete_frechet): Compute the FrÃ©chet distance between two polygonal curves in Euclidean space. ![GitHub stars](https://img.shields.io/github/stars/spiros/discrete_frechet?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.09254-yellowgreen?style=flat-square)
13. [docker-ukbiobank-utils](https://github.com/spiros/docker-ukbiobank-utils): Docker image for running UK Biobank utilities: ukbunpack, ukbfetch, ukblink, ukbgene, ukbmd5, ukbconv  ![GitHub stars](https://img.shields.io/github/stars/spiros/docker-ukbiobank-utils?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.22684-yellowgreen?style=flat-square)
14. [tofu](https://github.com/spiros/tofu): Tofu is a Python tool for generating synthetic UK Biobank data. ![GitHub stars](https://img.shields.io/github/stars/spiros/tofu?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.14048-yellowgreen?style=flat-square)
15. [ohdsi-etl-caliber](https://github.com/thehyve/ohdsi-etl-caliber): ETL scripts for caliber data ![GitHub stars](https://img.shields.io/github/stars/thehyve/ohdsi-etl-caliber?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.08313-yellowgreen?style=flat-square)
16. [phemap](https://github.com/spiros/phemap): Functions to map between ICD-10 terms and PheCodes for UK Biobank hospital electronic health records ![GitHub stars](https://img.shields.io/github/stars/spiros/phemap?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07918-yellowgreen?style=flat-square)
17. [ukbiobank-resources](https://github.com/spiros/ukbiobank-resources): A curated list for preprocessing, cleaning, mapping and analyzing UK Biobank data. ![GitHub stars](https://img.shields.io/github/stars/spiros/ukbiobank-resources?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11952-yellowgreen?style=flat-square)


### NATIONAL TEXT ANALYTICS (31)
1. [CogStack-NiFi](https://github.com/CogStack/CogStack-NiFi): Building data processing pipelines for documents processing with NLP using Apache NiFi and related services ![GitHub stars](https://img.shields.io/github/stars/CogStack/CogStack-NiFi?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.04379-yellowgreen?style=flat-square)
2. [CogStack-Pipeline](https://github.com/CogStack/CogStack-Pipeline): Distributed, fault tolerant batch processing for Natural Language Applications and Search, using remote partitioning ![GitHub stars](https://img.shields.io/github/stars/CogStack/CogStack-Pipeline?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.26282-yellowgreen?style=flat-square)
3. [CogStack-Recipes](https://github.com/CogStack/CogStack-Recipes): Example deployment recipes based on CogStack ![GitHub stars](https://img.shields.io/github/stars/CogStack/CogStack-Recipes?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02015-yellowgreen?style=flat-square)
4. [MedNorm Corpus](https://data.mendeley.com/datasets/b9x7xxb9sz/1): MedNorm is a corpus of 27,979 textual descriptions simultaneously mapped to both MedDRA and SNOMED-CT, sourced from five publicly available datasets across biomedical and social media domains.
The cross-terminology medical concept embeddings are 64-dimensional vectors for UMLS, MedDRA and SNOMED-CT concepts that are able to capture semantic similarities between concepts from different medical terminologies. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
5. [PheneBank- Processed Medline Abstracts and PMC full articles](https://zenodo.org/record/1197188#.X_Lw62T7Rqu): 24 million MEDLINE abstracts as well as 3.8M open-access PMC full articles annotated with 9 classes of entity: Phenotype, Disease, Anatomy, Cell, Cell_line, GPR, Gene_variant, Molecule, and Pathway mapped to five major ontologies: SNOMED, HPO, MeSH, PRO, and FMA. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
6. [ExECT-V2](https://github.com/swneurosci/ExECT-V2):  ![GitHub stars](https://img.shields.io/github/stars/swneurosci/ExECT-V2?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12737-yellowgreen?style=flat-square)
7. [MedCAT](https://github.com/CogStack/MedCAT): Medical Concept Annotation Tool ![GitHub stars](https://img.shields.io/github/stars/CogStack/MedCAT?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.35839-yellowgreen?style=flat-square)
8. [MedCATservice](https://github.com/CogStack/MedCATservice): Running MedCAT as a RESTful web service ![GitHub stars](https://img.shields.io/github/stars/CogStack/MedCATservice?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.14739-yellowgreen?style=flat-square)
9. [MedCATtrainer](https://github.com/CogStack/MedCATtrainer): A simple interface to inspect, improve and add concepts to biomedical NER+L -> MedCAT.  ![GitHub stars](https://img.shields.io/github/stars/CogStack/MedCATtrainer?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.33167-yellowgreen?style=flat-square)
10. [rake-nltk](https://github.com/csurfer/rake-nltk): Python implementation of the Rapid Automatic Keyword Extraction algorithm using NLTK. ![GitHub stars](https://img.shields.io/github/stars/csurfer/rake-nltk?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.15425-yellowgreen?style=flat-square)
11. [onset_pipeline](https://github.com/medesto/onset_pipeline):  ![GitHub stars](https://img.shields.io/github/stars/medesto/onset_pipeline?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02676-yellowgreen?style=flat-square)
12. [nlp2phenome](https://github.com/CogStack/nlp2phenome): using AI model to infer patient phenotypes from identified named entities (instances of biomedical concepts) ![GitHub stars](https://img.shields.io/github/stars/CogStack/nlp2phenome?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.13651-yellowgreen?style=flat-square)
13. [CogStack-SemEHR](https://github.com/CogStack/CogStack-SemEHR): Surfacing Semantic Data from Clinical Notes in Electronic Health Records for Tailored Care, Trial Recruitment and Clinical Research ![GitHub stars](https://img.shields.io/github/stars/CogStack/CogStack-SemEHR?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.16092-yellowgreen?style=flat-square)
14. [risk-score-builder](https://github.com/CogStack/risk-score-builder): Map clinical risk score concepts to structured definitions using ontologies, calculate structured risk scores from NLP. ![GitHub stars](https://img.shields.io/github/stars/CogStack/risk-score-builder?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.2489-yellowgreen?style=flat-square)
15. [CALIBER Drugdose](http://r-forge.r-project.org/R/?group_id=1598): medication dosage instructions in electronic health records are often in the form of text rather than numbers. This program is designed to convert the text into numbers for the dose, frequency, units, duration etc. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
16. [CRIS Natural Language Processing](https://maudsleybrc.nihr.ac.uk/facilities/clinical-record-interactive-search-cris/cris-natural-language-processing/): library of applications available for use within South London and Maudsley (SLaM) on the Clinical Record Interaction Search (CRIS) platform. Access to CRIS must be applied for in order to use applications. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
17. [EdIE-Viz](https://github.com/Edinburgh-LTG/edieviz/): provides an interface for stroke-related clinical concept recognition and negation detection in brain radiology reports. ![GitHub stars](https://img.shields.io/github/stars/Edinburgh-LTG/edieviz/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
18. [EdIE-R](https://github.com/Edinburgh-LTG/edieviz/tree/master/EdIE-R): a rule-based information extraction tool developed for brain imaging reports. ![GitHub stars](https://img.shields.io/github/stars/Edinburgh-LTG/edieviz/tree/master/EdIE-R?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
19. [EdIE-BiLSTM](https://github.com/Edinburgh-LTG/edieviz/tree/master/EdIE-N): a neural network system for named entity recognition and negation detection with a character-aware BiLSTM sentence encoder for brain imaging reports. ![GitHub stars](https://img.shields.io/github/stars/Edinburgh-LTG/edieviz/tree/master/EdIE-N?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
20. [EdIE-BERT](https://github.com/Edinburgh-LTG/edieviz/tree/master/EdIE-N): a neural network system for named entity recognition and negation detection with a pretrained BERT encoder (BlueBERT) for brain imaging reports. ![GitHub stars](https://img.shields.io/github/stars/Edinburgh-LTG/edieviz/tree/master/EdIE-N?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
21. [EndoMineR](https://github.com/sebastiz/EndoMineR/): a rule-based information extraction system for free-text and semi-structured endoscopy reports and their associated pathology specimens. ![GitHub stars](https://img.shields.io/github/stars/sebastiz/EndoMineR/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
22. [freetext-matching-algorithm](https://github.com/anoopshah/freetext-matching-algorithm): Source code for the Freetext Matching Algorithm, a natural language processing system for clinical text ![GitHub stars](https://img.shields.io/github/stars/anoopshah/freetext-matching-algorithm?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.06675-yellowgreen?style=flat-square)
23. [HELIN](https://github.com/cambridgeltl/HELIN): Demo Entity Linking API for the HDR Text Analytic Team. ![GitHub stars](https://img.shields.io/github/stars/cambridgeltl/HELIN?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11104-yellowgreen?style=flat-square)
24. [med7](https://github.com/kormilitzin/med7):  ![GitHub stars](https://img.shields.io/github/stars/kormilitzin/med7?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.14876-yellowgreen?style=flat-square)
25. [SIPHS](https://siphs.org/resources): a collection of software and datasets to support linguistic analysis of online health communities. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
26. [bioreddit](https://github.com/basaldella/bioreddit): Word embeddings trained on medical subreddits. ![GitHub stars](https://img.shields.io/github/stars/basaldella/bioreddit?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10375-yellowgreen?style=flat-square)
27. [BioWSD](http://staffwww.dcs.shef.ac.uk/people/M.Stevenson/resources/BioWSD/corpora/): corpora containing examples of two ambiguities from the biomedical domain (abbreviations and gene names). ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
28. [Cardiovascular research abstracts](http://staffwww.dcs.shef.ac.uk/people/M.Stevenson/resources/bio_contradictions/): corpus containing examples of potentially contradictory claims from Medline abstracts describing cardiovascular research intended as a useful resource for researchers working on similar problems. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
29. [cometa](https://github.com/cambridgeltl/cometa): Corpus of Online Medical EnTities: the cometA corpus ![GitHub stars](https://img.shields.io/github/stars/cambridgeltl/cometa?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10108-yellowgreen?style=flat-square)
30. [PheneBank](https://zenodo.org/record/1197188#.X9OFtcv7SjB): 4 million MEDLINE abstracts as well as 3.8M open-access PMC full articles annotated with 9 classes of entity - Phenotype, Disease, Anatomy, Cell, Cell_line, GPR, Gene_variant, Molecule, and Pathway mapped to five major ontologies - SNOMED, HPO, MeSH, PRO, and FMA. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
31. [SapBERT](https://huggingface.co/cambridgeltl/SapBERT-from-PubMedBERT-fulltext): Despite the widespread success of self-supervised learning via masked language models, learning representations directly from text to accurately capture complex and fine-grained semantic relationships in the biomedical domain remains as a challenge. SapBERT is a pre-training scheme based on BERT. It self-aligns the representation space of biomedical entities with a metric learning objective function leveraging UMLS, a collection of biomedical ontologies with >4M concepts. ![GitHub stars](https://img.shields.io/github/stars/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)


### UNCATEGORISED (67)
1. [20190927_IntroductionGithub_HDRUK](https://github.com/semacu/20190927_IntroductionGithub_HDRUK):  ![GitHub stars](https://img.shields.io/github/stars/semacu/20190927_IntroductionGithub_HDRUK?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05441-yellowgreen?style=flat-square)
2. [ADR-graph](https://github.com/KHP-Informatics/ADR-graph): Predicting adverse drug reactions from a knowledge graph ![GitHub stars](https://img.shields.io/github/stars/KHP-Informatics/ADR-graph?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07716-yellowgreen?style=flat-square)
3. [BRASS](https://github.com/cancerit/BRASS): Breakpoints via assembly - Identifies breaks and attempts to assemble rearrangements. ![GitHub stars](https://img.shields.io/github/stars/cancerit/BRASS?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.23126-yellowgreen?style=flat-square)
4. [Biomarker_Identification](https://github.com/jaw-bioinf/Biomarker_Identification): Biomarker Identification for Bravo, Williams, Gkoutos and Acharjee ![GitHub stars](https://img.shields.io/github/stars/jaw-bioinf/Biomarker_Identification?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
5. [CPRD_HES_variability](https://github.com/prockenschaub/CPRD_HES_variability):  ![GitHub stars](https://img.shields.io/github/stars/prockenschaub/CPRD_HES_variability?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05561-yellowgreen?style=flat-square)
6. [CPRD_UTI_sepsis_elderly](https://github.com/prockenschaub/CPRD_UTI_sepsis_elderly): This is the R code and code lists used to analyse the data for paper "Antibiotic prescribing for lower UTI in elderly patients in primary care and risk of bloodstream infection: A cohort study using electronic health records in England" ![GitHub stars](https://img.shields.io/github/stars/prockenschaub/CPRD_UTI_sepsis_elderly?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.09313-yellowgreen?style=flat-square)
7. [Pulse Wave Database](https://github.com/peterhcharlton/pwdb/): The aim of this project is to develop a database of simulated pulse waves with which to test pulse wave analysis algorithms.
A database of simulated pulse waves representative of a sample of healthy adults was created. It contains several types of pulse waves (such as those measured by smart watches and fitness trackers) at common measurement sites for 4,374 virtual subjects aged 25 - 75 years old. The virtual subjects exhibit a range of cardiovascular properties across the normal ranges encountered in healthy people. The results were verified against in vivo data. Case studies were published (at https://doi.org/10.1152/ajpheart.00218.2019 ) demonstrating the utility of the database.
The database is a useful resource for research and education on the topics of - signal processing, physiology, and computational modelling. ![GitHub stars](https://img.shields.io/github/stars/peterhcharlton/pwdb/?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
8. [DWGSIM](https://github.com/nh13/DWGSIM): Whole Genome Simulator for Next-Generation Sequencing ![GitHub stars](https://img.shields.io/github/stars/nh13/DWGSIM?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.33801-yellowgreen?style=flat-square)
9. [Data-Science-for-Docs](https://github.com/carpentries-incubator/Data-Science-for-Docs): Data Science For Practicing Clinicians ![GitHub stars](https://img.shields.io/github/stars/carpentries-incubator/Data-Science-for-Docs?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.27522-yellowgreen?style=flat-square)
10. [EHRtemporalVariability](https://github.com/hms-dbmi/EHRtemporalVariability): R package for delineating temporal dataset shifts in Eletronic Health Records ![GitHub stars](https://img.shields.io/github/stars/hms-dbmi/EHRtemporalVariability?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11582-yellowgreen?style=flat-square)
11. [EPACTS](https://github.com/statgen/EPACTS):  ![GitHub stars](https://img.shields.io/github/stars/statgen/EPACTS?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.17032-yellowgreen?style=flat-square)
12. [ESCs_models](https://github.com/MinhongW/ESCs_models): Agent-based modelling of pattern formation in embryonic stem cells ![GitHub stars](https://img.shields.io/github/stars/MinhongW/ESCs_models?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02077-yellowgreen?style=flat-square)
13. [Electronic-Monitoring-Device-Adherence-Typology](https://github.com/hollytibble/Electronic-Monitoring-Device-Adherence-Typology): Study Aim: To construct a data-driven multi-dimensional typology of medication non-adherence in children with asthma.   ![GitHub stars](https://img.shields.io/github/stars/hollytibble/Electronic-Monitoring-Device-Adherence-Typology?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05478-yellowgreen?style=flat-square)
14. [FM-summary](https://github.com/hailianghuang/FM-summary): Fine-mapping method only using summary statistics ![GitHub stars](https://img.shields.io/github/stars/hailianghuang/FM-summary?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
15. [HDR-UK_Webmapping_in_R](https://github.com/alalexiou/HDR-UK_Webmapping_in_R): How to use leaflet to make an interactive map using child obesity at reception year data in England. ![GitHub stars](https://img.shields.io/github/stars/alalexiou/HDR-UK_Webmapping_in_R?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05435-yellowgreen?style=flat-square)
16. [HDRUK-fellows-day-2018](https://github.com/alastair-droop/HDRUK-fellows-day-2018): Machine Learning Resources collected from the HDRUK UK Fellows's day 2018 (2018-11-15) ![GitHub stars](https://img.shields.io/github/stars/alastair-droop/HDRUK-fellows-day-2018?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
17. [KGPhenotyping](https://github.com/Honghan/KGPhenotyping): knowledge graph based phenotyping ![GitHub stars](https://img.shields.io/github/stars/Honghan/KGPhenotyping?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11558-yellowgreen?style=flat-square)
18. [MetaXcan](https://github.com/hakyimlab/MetaXcan): MetaXcan software and manuscript ![GitHub stars](https://img.shields.io/github/stars/hakyimlab/MetaXcan?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.3652-yellowgreen?style=flat-square)
19. [PGM](https://github.com/azizfurqan/PGM):  ![GitHub stars](https://img.shields.io/github/stars/azizfurqan/PGM?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02077-yellowgreen?style=flat-square)
20. [RDMP](https://github.com/HicServices/RDMP): Research Data Management Platform (RDMP) is an open source application for the loading,linking,anonymisation and extraction of datasets stored in relational databases. ![GitHub stars](https://img.shields.io/github/stars/HicServices/RDMP?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.49929-yellowgreen?style=flat-square)
21. [RF-FeatureSelection-PowerAnalysis](https://github.com/joelarkman/RF-FeatureSelection-PowerAnalysis):  ![GitHub stars](https://img.shields.io/github/stars/joelarkman/RF-FeatureSelection-PowerAnalysis?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
22. [ScienceCOPEMethodsCode](https://github.com/MGHcteu/ScienceCOPEMethodsCode): Code accompanying map generation in Drew et al. Science manuscript ![GitHub stars](https://img.shields.io/github/stars/MGHcteu/ScienceCOPEMethodsCode?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.03876-yellowgreen?style=flat-square)
23. [artificialMHR](https://github.com/KCL-Health-NLP/artificialMHR):  ![GitHub stars](https://img.shields.io/github/stars/KCL-Health-NLP/artificialMHR?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02664-yellowgreen?style=flat-square)
24. [bhpm](https://github.com/rcarragh/bhpm): Bayesian Hierarchical Poisson Models for Multiple Grouped Outcomes with Clustering ![GitHub stars](https://img.shields.io/github/stars/rcarragh/bhpm?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.06759-yellowgreen?style=flat-square)
25. [cCFRDiamondPrincess](https://github.com/thimotei/cCFRDiamondPrincess):  ![GitHub stars](https://img.shields.io/github/stars/thimotei/cCFRDiamondPrincess?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.06617-yellowgreen?style=flat-square)
26. [cardio](https://github.com/Nico-Curti/cardio): Pulse oximetry data processing and classification ![GitHub stars](https://img.shields.io/github/stars/Nico-Curti/cardio?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12533-yellowgreen?style=flat-square)
27. [cgpBattenberg](https://github.com/cancerit/cgpBattenberg): Battenberg algorithm and associated implementation script ![GitHub stars](https://img.shields.io/github/stars/cancerit/cgpBattenberg?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.24794-yellowgreen?style=flat-square)
28. [clockwork](https://github.com/iqbal-lab-org/clockwork): CRyPTIC data processing pipelines ![GitHub stars](https://img.shields.io/github/stars/iqbal-lab-org/clockwork?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.28071-yellowgreen?style=flat-square)
29. [clustering_w2v](https://github.com/RichJackson/clustering_w2v):  ![GitHub stars](https://img.shields.io/github/stars/RichJackson/clustering_w2v?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.03164-yellowgreen?style=flat-square)
30. [dpclust](https://github.com/Wedge-lab/dpclust): Dirichlet Process based methods for subclonal reconstruction of tumours ![GitHub stars](https://img.shields.io/github/stars/Wedge-lab/dpclust?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.18415-yellowgreen?style=flat-square)
31. [drug_target_lof](https://github.com/ericminikel/drug_target_lof): Evaluating potential drug targets through human loss-of-function genetic variation ![GitHub stars](https://img.shields.io/github/stars/ericminikel/drug_target_lof?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0777-yellowgreen?style=flat-square)
32. [hdr_uk_demo](https://github.com/bwpapiez/hdr_uk_demo): HDR UK Summer School demo for Image Registration ![GitHub stars](https://img.shields.io/github/stars/bwpapiez/hdr_uk_demo?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
33. [hdruk-phd-handbook](https://github.com/cwcyau/hdruk-phd-handbook): Handbook for the HDRUK-Turing PhD Programme in Health Data Science ![GitHub stars](https://img.shields.io/github/stars/cwcyau/hdruk-phd-handbook?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.15812-yellowgreen?style=flat-square)
34. [hdruk_bbmri_hack](https://github.com/simonjupp/hdruk_bbmri_hack):  ![GitHub stars](https://img.shields.io/github/stars/simonjupp/hdruk_bbmri_hack?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
35. [hdruk_summerschool_session1](https://github.com/athro/hdruk_summerschool_session1): EHR analysis for the HDR UK summer school 2019 ![GitHub stars](https://img.shields.io/github/stars/athro/hdruk_summerschool_session1?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
36. [hdruk_summerschool_session_1_2](https://github.com/athro/hdruk_summerschool_session_1_2): Docker image for first two sessions at the HDRUK summer school 2019 ![GitHub stars](https://img.shields.io/github/stars/athro/hdruk_summerschool_session_1_2?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
37. [hilton-keeling-estimating-R0](https://github.com/JBHilton/hilton-keeling-estimating-R0): The code in this repository accompanies our upcoming paper "Estimation of country-level basic reproductive ratios for novelCoronavirus (COVID-19) using synthetic contact matrices". A preprint version of this paper is available at https://www.medrxiv.org/content/10.1101/2020.02.26.20028167v1. Some of our notation has changed between the preprint and the submission version of the paper. In particular, the age-specific susceptibility is now denoted by sigma rather than z. ![GitHub stars](https://img.shields.io/github/stars/JBHilton/hilton-keeling-estimating-R0?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02669-yellowgreen?style=flat-square)
38. [ldsc](https://github.com/bulik/ldsc): LD Score Regression (LDSC) ![GitHub stars](https://img.shields.io/github/stars/bulik/ldsc?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.35139-yellowgreen?style=flat-square)
39. [loftee](https://github.com/konradjk/loftee):  ![GitHub stars](https://img.shields.io/github/stars/konradjk/loftee?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.30803-yellowgreen?style=flat-square)
40. [lrTaps](https://github.com/jfeicheng92/lrTaps):  ![GitHub stars](https://img.shields.io/github/stars/jfeicheng92/lrTaps?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.14917-yellowgreen?style=flat-square)
41. [manuscript_code](https://github.com/bloodcellgwas/manuscript_code): Manuscript code for our paper: bit.ly/bloodcellgwas ![GitHub stars](https://img.shields.io/github/stars/bloodcellgwas/manuscript_code?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10517-yellowgreen?style=flat-square)
42. [multiomic_AITD](https://github.com/TiphaineCMartin/multiomic_AITD): Scripts associated with the paper doi: https://doi.org/10.1101/662957 ![GitHub stars](https://img.shields.io/github/stars/TiphaineCMartin/multiomic_AITD?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.06436-yellowgreen?style=flat-square)
43. [mvGWAMA](https://github.com/Kyoko-wtnb/mvGWAMA): Python script for multivariate GWAS meta-analysis ![GitHub stars](https://img.shields.io/github/stars/Kyoko-wtnb/mvGWAMA?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.08473-yellowgreen?style=flat-square)
44. [ontologies](https://github.com/biosemantics/ontologies):  ![GitHub stars](https://img.shields.io/github/stars/biosemantics/ontologies?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.04441-yellowgreen?style=flat-square)
45. [predCAN](https://github.com/bio-ontology-research-group/predCAN): Ontology-based prediction of cancer driver genes ![GitHub stars](https://img.shields.io/github/stars/bio-ontology-research-group/predCAN?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02866-yellowgreen?style=flat-square)
46. [komenti](https://github.com/reality/komenti): Semantic query and text mining tool ![GitHub stars](https://img.shields.io/github/stars/reality/komenti?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.20536-yellowgreen?style=flat-square)
47. [generate-patients](https://github.com/reality/generate-patients):  ![GitHub stars](https://img.shields.io/github/stars/reality/generate-patients?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
48. [add_synonyms](https://github.com/reality/add_synonyms): add a synonym vocabulary to an ontology ![GitHub stars](https://img.shields.io/github/stars/reality/add_synonyms?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.01915-yellowgreen?style=flat-square)
49. [sampledd](https://github.com/reality/sampledd): Sample MIMIC patients who die during their stay, and associated text records ![GitHub stars](https://img.shields.io/github/stars/reality/sampledd?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02077-yellowgreen?style=flat-square)
50. [mimpred](https://github.com/reality/mimpred): (In development) Reproducible experimental foundation for experiments with MIMIC ![GitHub stars](https://img.shields.io/github/stars/reality/mimpred?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
51. [mim3val](https://github.com/reality/mim3val): mimic3 nlp evaluation. needs a NOTEEVENTS.csv, processed files from Komenti ![GitHub stars](https://img.shields.io/github/stars/reality/mim3val?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.03217-yellowgreen?style=flat-square)
52. [mimsim](https://github.com/reality/mimsim): Tools for performing semantic similarity over MIMIC-III ![GitHub stars](https://img.shields.io/github/stars/reality/mimsim?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0-yellowgreen?style=flat-square)
53. [ECMO](https://github.com/reality/ECMO): Extended Clinical Measurement Ontology ![GitHub stars](https://img.shields.io/github/stars/reality/ECMO?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.04924-yellowgreen?style=flat-square)
54. [synonym_expansion_validation](https://github.com/reality/synonym_expansion_validation):  ![GitHub stars](https://img.shields.io/github/stars/reality/synonym_expansion_validation?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05425-yellowgreen?style=flat-square)
55. [UNMIREOT](https://github.com/bio-ontology-research-group/UNMIREOT): Tools to identify, diagnose, and semi-automatically repair hidden contradictions in biomedical ontologies ![GitHub stars](https://img.shields.io/github/stars/bio-ontology-research-group/UNMIREOT?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11155-yellowgreen?style=flat-square)
56. [ocimido](https://github.com/sap218/ocimido): An ontology for ocular immune-mediated inflammatory diseases ![GitHub stars](https://img.shields.io/github/stars/sap218/ocimido?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.10464-yellowgreen?style=flat-square)
57. [jabberwocky](https://github.com/sap218/jabberwocky): toolkit for those nonsensical ontologies ![GitHub stars](https://img.shields.io/github/stars/sap218/jabberwocky?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.09775-yellowgreen?style=flat-square)
58. [resource_manager](https://github.com/gkoutos-group/resource_manager): This is a simple resource manager ![GitHub stars](https://img.shields.io/github/stars/gkoutos-group/resource_manager?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07197-yellowgreen?style=flat-square)
59. [ringbp](https://github.com/cmmid/ringbp):  ![GitHub stars](https://img.shields.io/github/stars/cmmid/ringbp?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.05316-yellowgreen?style=flat-square)
60. [software_differences_paper](https://github.com/ebmdatalab/software_differences_paper):  ![GitHub stars](https://img.shields.io/github/stars/ebmdatalab/software_differences_paper?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.0583-yellowgreen?style=flat-square)
61. [treeImbalance](https://github.com/bdearlove/treeImbalance): An R package for detecting asymmetry in time-sampled phylogenetic trees. ![GitHub stars](https://img.shields.io/github/stars/bdearlove/treeImbalance?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.03721-yellowgreen?style=flat-square)
62. [treeSeg](https://github.com/merlebehr/treeSeg): Implementation of the treeSeg algorithm as an R package. ![GitHub stars](https://img.shields.io/github/stars/merlebehr/treeSeg?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.11282-yellowgreen?style=flat-square)
63. [treeseq-inference](https://github.com/mcveanlab/treeseq-inference): Work for the tree sequence inference paper. ![GitHub stars](https://img.shields.io/github/stars/mcveanlab/treeseq-inference?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.21308-yellowgreen?style=flat-square)
64. [ASSIGN](https://github.com/endeavourhealth-discovery/ASSIGN):  ![GitHub stars](https://img.shields.io/github/stars/endeavourhealth-discovery/ASSIGN?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.12986-yellowgreen?style=flat-square)
65. [uvR](https://github.com/markocherrie/uvR): R package to support analysing UVR data ![GitHub stars](https://img.shields.io/github/stars/markocherrie/uvR?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02041-yellowgreen?style=flat-square)
66. [vascular-ageing](https://github.com/LorenzoDallOlio/vascular-ageing): Container for Python analysis codes regarding "Prediction of vascular ageing based on smartphoneacquired PPG signals" ![GitHub stars](https://img.shields.io/github/stars/LorenzoDallOlio/vascular-ageing?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.02077-yellowgreen?style=flat-square)
67. [vec2sparql](https://github.com/gkoutos-group/vec2sparql): SPARQL Endpoint with functions for computing embedding similarities ![GitHub stars](https://img.shields.io/github/stars/gkoutos-group/vec2sparql?style=flat-square) ![Criticality Score](https://img.shields.io/badge/criticality--score-0.07847-yellowgreen?style=flat-square)


> **Note:** Contributions are noted where there are direct or indirect contributions (funding and support) made by HDR UK to the above projects.

---
### How to contribute OSS HDR UK projects

1. Create a [Github.com account](https://github.com/join) if you do not have one already
2. Create a [fork of this project](https://github.com/HDRUK/oss).
3. Edit the [oss_projects.yml](data/oss_projects.yml) file and add an entry for your project using this template:
   ```yaml
   - name: 'Project Awesome'
     description: 'Describe what your project is about'
     url: 'https://github.com/YOUREPO (Provide your Github Repo URL)'
     keywords:
      - 'example'
      - 'keyword'
     categories:
      - 'HDR UK'
      - 'themes'
   ```
4. Commit the changes to your fork and [submit a Pull Request](https://github.com/HDRUK/oss/compare) against the `master` branch of this project.

We'll review all requests and accept them according to HDR UK's policies. If accepted, your project will be listed above
